woocommerce/woocommerce-admin
# Create reusable E2E testing environment package.

This PR seeks to introduce a new package, `@woocommerce/e2e-env` which can be used to bootstrap an End-to-End testing environment for WooCommerce and extensions.

### Detailed test instructions:

- Review instructions in `packages/e2e-env/README.md`
- Follow usage instructions in the `woocommerce-admin` repo
- Test in the `woocommerce-gutenberg-products-block` repo:
- Check out https://github.com/woocommerce/woocommerce-gutenberg-products-block/compare/try/e2e-package?expand=1
- Copy all files from `packages/e2e-env` to `node_modules/@woocommerce/e2e-env`
